pygmt/src/xyz2grd.py

```diff
@@ -14,14 +14,18 @@
 
 @fmt_docstring
 @use_alias(
+    A="duplicate",
+    D="dname",
```

Comment:
I don't think `dname` is a good name.

Reply:
I just pulled it directly from the GMT docs. What would you recommend instead?

Reply:
I don't have a good name to recommend. I just feel that it's difficult to guess what the parameter does from the name `dname`.

Reply:
It seems to me that xyz2grd -D does the same thing as `grdedit` in setting the header information. Related to past conversations about -X and -Y versus shift_origin, would it be better to leave this out of xyz2grd and instead add a wrapper for grdedit in order to only have one way to do this task?

Reply:
removed in d890a75